.github/workflows/main.yml

```
@@ -57,12 +66,21 @@ jobs:
       - name: Build gem
         run: gem build
 
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: ${{ vars.AWS_REGION }}
+          role-to-assume: ${{ vars.AWS_IAM_ROLE_ARN }}
+      - uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          parse-json-secrets: true
+          secret-ids: RUBYGEMS,${{ vars.RUBYGEMS_SECRET_ARN }}
```

@stoyanzhekov The env var is referenced as `RUBYGEMS_TOKEN`.

```suggestion
          secret-ids: RUBYGEMS_TOKEN,${{ vars.RUBYGEMS_SECRET_ARN }}
```